Jenkinsfile

```diff
@@ -113,8 +113,14 @@ Note this is another run, will double the time and no guaranty to have same erro
 
     post {
         always {
-            archiveArtifacts(artifacts: 'notebooks/*.ipynb, pavics-sdi-*/docs/source/notebooks/*.ipynb, finch-*/docs/source/notebooks/*.ipynb, raven-*/docs/source/notebooks/*.ipynb, esgf-compute-api-*/examples/*.ipynb, PAVICS-landing-*/content/notebooks/climate_indicators/*.ipynb, buildout/*.output.ipynb, buildout/env-dump/',
-                             fingerprint: true)
+            archiveArtifacts(artifacts: 'notebooks/*.ipynb', fingerprint: true)
+            archiveArtifacts(artifacts: 'pavics-sdi-*/docs/source/**/*.ipynb', fingerprint: true, excludes: 'pavics-sdi-*/docs/source/deprecated/**/*')
```

Comment:
I would prefer we explicitly add the notebooks we actually test to avoid "surprise".

This regex will also collect 
```
./tutorials/bias_correction.ipynb
./projects/notebooks/cruesdeca.ipynb
``` 
which we do not test.

Reply:
Done.